Makefile

```diff
@@ -1,3 +1,17 @@
+###############################################################################
+# For local builds, YAML_CMD can be made faster by running
+# "go get github.com/mikefarah/yaml" and changing YAML_CMD to "yaml"
```

Comment:
>and changing YAML_CMD to "yaml"

Is that part necessary? Shouldn't `shell which yaml` evaluate correctly without needing to pass in `YAML_CMD`? And if so, can't we use `:=` instead of `?=`?

Reply:
good catch - this can be fixed in calico_node/Makefile as well.